scripts/high_level_motion_unit_test.py

```diff
@@ -0,0 +1,173 @@
+#! /usr/bin/env python
+import sys
+import rospy
+from iarc7_msgs.msg import OdometryArray
+import actionlib
+import tf2_ros
+from iarc7_motion.msg import QuadMoveGoal, QuadMoveAction
+from iarc7_safety.SafetyClient import SafetyClient
+
+def test():
+    safety_client = SafetyClient('high_level_motion_unit_test')
```

Comment:
Name should get changed here

Reply:
fixed, changed to `invalid_task_transition_unit_test`